userdocs/profiles.dev/docusaurus.config.js

```diff
@@ -50,11 +50,11 @@ module.exports = {
             },
             {
               label: 'Developer Docs: Profile User',
-              to: '/docs/intro/simple-install',
+              to: '/docs/installer-docs/simple-install',
             },
             {
               label: 'Developer Docs: Catalog Manager',
-              to: '/docs/intro/simple-catalog',
+              to: '/docs/catalog-docs/simple-catalog',
```

Comment:
@Callisto13 this was causing compilation errors during `yarn deploy` as the links are dead. I think these are the correct new links? lmk

Reply:
yeh think so